paramiko/client.py

```diff
@@ -426,6 +426,7 @@ def exec_command(
         command,
         bufsize=-1,
         timeout=None,
+        open_channel_timeout=None,
```

Comment:
I think that the new kwarg should go last, and then in the next major version we can re-order and make kwarg-only (for example https://github.com/ploxiln/paramiko-ng/pull/120)

What do you think about a shorter argument name, maybe "open_timeout"?

Reply:
Sounds good - will move it to the end & call it *open_timeout*